flow.js

```diff
@@ -31,7 +31,10 @@ class Flow {
         {condition: 'livingAreas', nextStep: 'repair-living-areas-problems'},
         {condition: 'outside', nextStep: 'repair-outside-problems'}
       ]},
-      'repair-outside-problems': { prevSteps: '', nextStep:'repair-description'},
+      'repair-outside-problems': { prevSteps: '', nextStep: [
+        {condition: 'outdoorSecurityLights', nextStep: 'repair-description'},
+        {condition: 'garageRoofAndDoor', nextStep: 'repair-garage-problems'}]},
```

Comment:
This issue is described as "Garage, including roof and door" and includes "lock damage" and "broken into" too.
I think it would be best to make the key less specific, i.e.
```suggestion
        {condition: 'garage', nextStep: 'repair-garage-problems'}]},
```